liquigraph-cli/src/main/java/org/liquigraph/cli/LiquigraphCli.java

```diff
@@ -73,6 +73,12 @@
     )
     private String graphDbUri;
 
+    @Parameter(
+            names = {"--database", "-u"},
```

Comment:
small typo, the short option cannot be `-u` here as `-u` is the username option alias